tasks/create-pyxis-image/tests/test-create-pyxis-image-multi-containerimages-one-arch.yaml

```diff
@@ -72,10 +72,16 @@ spec:
       runAfter:
         - setup
     - name: check-result
+      params:   
+        - name: pyxisDataPath
+          value: $(tasks.run-task.results.pyxisDataPath)
       workspaces:
-        - name: data
+        - name: data 
```

Comment:
Extra trailing space.

Reply:
That's what I get for copy pasting. Fixed